modules/specialized-guides/pages/large-deployments/hub-online-sync.adoc

```diff
@@ -0,0 +1,213 @@
+. In the field [literal]``Token`` paste the token that was created on the peripheral server.
+. In the field [literal]``Root CA certificate`` specify the certificate using one of the options:
+  * Use option [literal]``Not needed`` if both hub and peripheral servers have the same certificate authority.
+  * Use option [literal]``Upload a file`` if the servers have different certificate authorities to upload a certificate file.
+  * Use option [literal]``Paste a PEM certificate`` to paste a certificate.  
+. Click button btn:[Register].
+  A newly registered peripheral serevr will appear on screen [literal]``Peripherals Configuration``.
+
+
+
+=== Registration from the hub server directly
+
+It is possible to initiate the registration of a peripheral server from hub server, without any interaction with the peripheral server.
+
+[[direct-registration]]
+.Procedure: Direct registering from the hub server
+
+. On the hub server, go to menu:Hub Configuration[Peripherals Configuration].
+. Click button btn:[Add peripheral].
+  A new form [literal]``Register a new peripheral server`` opens.
+. In the field [literal]``Peripheral Server FQDN`` enter the name of the peripheral server.
+. In the field [literal]``Registration mode`` select option [literal]``Administrator User/Password``.
+. In the fields [literal]``Username`` and [literal]``Password`` enter the credentials for the peripheral server.
+. In the field [literal]``Root CA certificate`` specify the certificate using one of the options:
+  * Use option [literal]``Not needed`` if both hub and peripheral servers have the same certificate authority.
+  * Use option [literal]``Upload a file`` if the servers have different certificate authorities to upload a certificate file.
+  * Use option [literal]``Paste a PEM certificate`` in cases when PEM certificate is used.
+. Click button btn:[Register].
+. The newly reigistered peripheral server will be shown in the menu:Systems[System List] with the value [literal]``Foreign`` in the column [literal]``System Type``.
+. To access its details, click on the peripheral server's name and select tab menu:Details[Peripheral Server].
+
+Peripheral server uses hub to access the vendor channels and does not connect to the {scclongform} directly.
+Therefore, if you open configured peripheral server's page menu:Admin[Setup Wizard > Organization Credentials], menu:Admin[Setup Wizard > Products] or menu:Admin[Setup Wizard > PAYG Connections], you will see a notification that this is peripheral server and its connections are managed via hub.
+
+
+=== Access tokens
+
+All existing tokens are shown in menu:Hub Configuration[Access Tokens].
+
+A token can be [literal]``Consumed`` and [literal]``Issued``.
+
+Consumed::
+This type of token is generated on the peripheral server and received by the hub server to be used
+
+Issued::
+This type of token is issued by the hub server to be used by the peripheral server.
```

Comment:
This is written from the perspective of the hub. 

The description would be mirrored if looking at the same UI page on a peripheral. In that case, issued tokens are the one generated by the peripheral and sent to the hub, while consumed are those issued by the hub and used by the peripheral to establish a connection.

Reply:
Thank you @mackdk.